client/src/components/App.tsx

```diff
@@ -113,13 +115,12 @@ function App () {
         </Navbar>
         <Container style={{ marginTop: '200px' }}>
           <Routes>
-            <Route exact path='/' element={<PrivateRoute />} />
+            <Route path='/' element={<PrivateRoute />} />
```

Comment:
Does removing `exact` not break all the other routes? Since it matches them in the order it finds them?

Reply:
no, it makes an error and after some googling, react route v6 doesnt support exact, its from <v6. it matches exact by default